.github/workflows/python.yml

```diff
@@ -107,81 +107,49 @@ jobs:
         instance: [0, 1, 2]
 
     env:
-      SENTRY_PYTHON3: 1
-      PIP_DISABLE_PIP_VERSION_CHECK: on
-
-      SENTRY_LIGHT_BUILD: 1
-      SENTRY_SKIP_BACKEND_VALIDATION: 1
       MIGRATIONS_TEST_MIGRATE: 1
-      PYTEST_SENTRY_DSN: https://6fd5cfea2d4d46b182ad214ac7810508@sentry.io/2423079
-      PYTEST_ADDOPTS: "--reruns 3"
-      PYTEST_SENTRY_ALWAYS_REPORT: 1
-
-      # services configuration
-      SENTRY_REDIS_HOST: redis
-      DATABASE_URL: postgresql://postgres:postgres@localhost/sentry
 
       # Number of matrix instances
       TOTAL_TEST_GROUPS: ${{ strategy.job-total }}
 
     steps:
-      - name: Install System Dependencies
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y --no-install-recommends \
-            libxmlsec1-dev \
-            libmaxminddb-dev
-
       - uses: actions/checkout@v2
 
-      - name: Set up outputs
-        id: config
-        env:
-          MATRIX_INSTANCE: ${{ matrix.instance }}
+      - uses: volta-cli/action@v1
+
+      - name: Set python version output
+        id: python-version
         run: |
           echo "::set-output name=python-version::$(awk 'FNR == 2' .python-version)"
-          echo "::set-output name=matrix-instance-number::$(($MATRIX_INSTANCE+1))"
 
-      - name: Set up Python ${{ steps.config.outputs.python-version }}
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ steps.config.outputs.python-version}}
 
-      - name: Install pip
-        run: |
-          pip install --no-cache-dir --upgrade "pip>=20.0.2"
+      # Until GH composite actions can use `uses`, we need to setup python here
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ steps.python-version.outputs.python-version }}
 
-      - name: Get pip cache dir
-        id: pip-cache
-        run: |
-          echo "::set-output name=dir::$(pip cache dir)"
+      - name: Setup sentry env
+        uses: ./.github/actions/setup-sentry
+        id: setup
+        with:
+          python: 3
 
       - name: pip cache
         uses: actions/cache@v1
         with:
-          path: ${{ steps.pip-cache.outputs.dir }}
+          path: ${{ steps.setup.outputs.pip-cache-dir }}
           key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements-*.txt') }}
           restore-keys: |
             ${{ runner.os }}-pip-
 
-      - name: Install Python Dependencies
-        env:
-          PGPASSWORD: postgres
+      - name: Install rb
```

Comment:
We don't need to override rb anymore, I cleaned it up in a commit somewhere but I think it got lost in a merge or w/e.